### PR TITLE
Added the option to add labels to service monitor

### PR DESCRIPTION
**Pull request by ofirshtrull**
I added this because my prom operator is searching for a specific label "release: prometheus-operator" when it searches for service monitors between namespaces
so by this, every developer can add a specific label according to his needs
also bumped the version of the helm for such a change

### CODE REVIEW

File: helm/vernemq/README.md [hunk 1]
```diff
@@ -104,8 +104,9 @@ Parameter | Description | Default
 `statefulset.podManagementPolicy` | start and stop pods in Parallel or OrderedReady (one-by-one.)  **Note** - Cannot change after first release. | `OrderedReady`
 `statefulset.terminationGracePeriodSeconds` | configure how much time VerneMQ takes to move offline queues to other nodes | `60`
 `statefulset.updateStrategy` | Statefulset updateStrategy | `RollingUpdate`
-`statefulset.lifecycle | Statefulset lifecycle hooks | `{}`
+`statefulset.lifecycle` | Statefulset lifecycle hooks | `{}`
 `serviceMonitor.create` | whether to create a ServiceMonitor for Prometheus Operator | `false`
+`serviceMonitor.labels` | whether to add omre labels to ServiceMonitor for Prometheus Operator | `{}`
```
Comment (angeloskaltsikis) [inline helm/vernemq/README.md:109]: Hey @ofirshtrull, I noticed a small typo in the description. Please change `omre` -> `more`

Comment (ofirshtrull) [reply helm/vernemq/README.md:109]: Fixed

**Comment by ofirshtrull** [comment]
@ioolkos can you please push to the helm repo the new version?

**Comment by ofirshtrull** [comment]
@ioolkos ??

**Pull request merged**